main.nf

```diff
@@ -1,41 +1,48 @@
 #!/usr/bin/env nextflow
+
 /*
 vim: syntax=groovy
 -*- mode: groovy;-*-
 ========================================================================================
-                        N G I    S M A L L    R N A - S E Q
+          N G I    S M A L L    R N A - S E Q    B E S T    P R A C T I C E
 ========================================================================================
- miRNA  Analysis Pipeline. Started May 2016.
+ small RNA-seq  Analysis Pipeline. Started May 2016.
  @Authors
  Phil Ewels <phil.ewels@scilifelab.se>
+ Chuan Wang <chuan.wang@scilifelab.se>
  Rickard Hammarén <rickard.hammaren@scilifelab.se>
 ----------------------------------------------------------------------------------------
  Basic command:
  $ nextflow main.nf
- 
+
  Pipeline variables can be configured with the following command line options:
- --genome [GRCh37 | GRCm38]
- --index [path to STAR index]
+ --genome [ID]
+ --index [path to bowtie2 index] (alternative  to --genome)
  --gtf [path to GTF file]
- --files [path to input files]
- 
+ --reads [path to input files]
+
  For example:
- $ nextflow rnaseq.nf --files path/to/data/*fq.gz
+ $ nextflow rnaseq.nf --reads path/to/data/*fq.gz
 ----------------------------------------------------------------------------------------
  Pipeline overview:
- - FastQC - read quility control
- - cutadapt - trimming
- - Bowtie 1 - align against miRBase mature
- - Samtools idxstats - count mature miRBase alignments
- - Bowtie 1 - align against miRBase hairpin (mature unaligned only)
- - Samtools idxstats - count hairpin miRBase alignments
- - Bowtie 2 - align against host whole genome
- - Bamutils - filter alignments by length
- - featureCounts - count alignments
- - edgeR count normalisation and QC
-   - MDS plot clustering samples
-   - Heatmap of sample similarities
- - MultiQC
+ - 1:   FastQC for raw sequencing reads quility control
+ - 2:   Trim Galore! for adapter trimming
+ - 3.1: Bowtie 1 alignment against miRBase mature miRNA
+ - 3.2: Post-alignment processing of miRBase mature miRNA counts
+ - 3.3: edgeR analysis on miRBase mature miRNA counts
+        - TMM normalization and a table of top expression mature miRNA
+        - MDS plot clustering samples
+        - Heatmap of sample similarities
+ - 4.1: Bowtie 1 alignment against miRBase hairpin for the unaligned reads in step 3
+ - 4.2: Post-alignment processing of miRBase hairpin counts
+ - 4.3: edgeR analysis on miRBase hairpin counts
+        - TMM normalization and a table of top expression hairpin
+        - MDS plot clustering samples
+        - Heatmap of sample similarities
+ - 5.1: Bowtie 2 alignment against host reference genome
+ - 5.2: Post-alignment processing of Bowtie 2
+ - 6:   NGI-Visualization of Bowtie 2 alignment statistics
```

Comment:
Nice annotation of pipeline steps 👍 

Maybe `6` should be `5.3` as it's specific to bt2?